src/_base/harness/attributes/docker-base.yml

```diff
@@ -153,6 +158,9 @@ attributes:
           metricsEnabled: true
           metricsEndpoints:
             - port: php-fpm-metrics
+        tideways:
```

Comment:
I don't know if it's a good idea to implicitly enable tideways for all environments just because it's being used locally. Making it an explicit choice doesn't sound too bad

Reply:
there's a cost factor as well right? /cc @rgpjones

Reply:
Updated :+1: